pkg/cluster/resources.go

```diff
@@ -142,3 +142,33 @@ func CreateNamespace(cli client.Client, namespace string, metaOptions ...MetaOpt
 
 	return foundNamespace, nil
 }
+
+// CreateClusterRole creates cluster role required by model registry component.
```

Comment:
It's a generic function and others might need it to :) Thus a statement about the particular need is not needed, unless we are not allowed to touch it ;)

```suggestion
// CreateClusterRole creates cluster role based on define PolicyRules and optional metadata fields.
```